ros2/src/airsim_ros_pkgs/src/airsim_ros_wrapper.cpp

```diff
@@ -1404,6 +1429,48 @@ std::shared_ptr<sensor_msgs::msg::CompressedImage> AirsimROSWrapper::get_jpeg_ms
     return jpeg_message;
 }
 
+std::shared_ptr<image_metadata::Metadata> AirsimROSWrapper::get_jpeg_msg_metadata()
+{
+    std::string vehicle_name = "Copter";
+    auto height_data = airsim_client_->getDistanceSensorData("Distance1", vehicle_name);
+    auto distance_data = airsim_client_->getDistanceSensorData("Distance2", vehicle_name);
+    auto gps_data = airsim_client_->getGpsData("Gps", vehicle_name);
+
+    auto vehicle_attitude = get_camera_attitude_euler_msg(vehicle_name, "front_center_fixed");
+    auto gimbal_attitude = get_camera_attitude_euler_msg(vehicle_name, gimbal_vehicle_reference_camera_name_);

```

Comment:
so, if i'm trying to create a global tf matrix, do i only use the gimbal attitude or gimbal and vehicle both?

Reply:
I don't understand the question.  Both are global.